.circleci/config.yml

```diff
@@ -0,0 +1,68 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10.15.1
+    steps:
+      - checkout
+      - run:
+          command: npm ci
+      - run:
+          command: npm run build
+      - persist_to_workspace:
+          root: .
+          paths:
+            - build
+
+  deploy:
+    docker:
+      - image: olizilla/ipfs-dns-deploy
+        environment:
+          DOMAIN: webui.ipfs.io
+          BUILD_DIR: build
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - run:
+          name: Pin website, post notification for PRs or update DNS on master
+          command: |
+            # Upload build dir to cluster.ipfs.io
+            hash=$(ipfs-cluster-ctl \
+              --host /dnsaddr/cluster.ipfs.io \
+              --basic-auth $CLUSTER_USER:$CLUSTER_PASSWORD \
+              add --rmin 3 --rmax 3 --name "$DOMAIN" \
+              --recursive /tmp/workspace/$BUILD_DIR | tail -n1 | cut -d " " -f 2 )
```

Comment:
You want to use the `--quietest` or `--Q` option which only outputs the final hash, rather than `tail ...`